styleguide/template.md

```diff
@@ -0,0 +1,340 @@
+
+- Example: [Readme file](../readme.md)
+- Example: [Welcome to .NET](../docs/welcome.md)
+
+To link to a header in a Markdown file in the same repo, use relative linking + hashtag linking.
+
+- Example: [.NET Community](../docs/welcome.md#community)
+
+### External Links
+
+To link to an external file, use the full URL as the link.
+
+- Example: [GitHub](http://www.github.com)
+
+If a URL appears in a Markdown file, it will be transformed into a clickable link.
+
+- Example: http://www.github.com
+
+### Links to APIs
+
+The build system has some extensions that allow us to link to .NET Core APIs without having to use external links.  
+When linking to an API, you can use its unique identifier (UID) that is auto-generated from the source code.
+
+You can use one of the following syntax:
+1. Markdown link: `[link_text](xref:UID)`
+2. Auto link: `<xref:UID>`
+3. Shorthand form: `@UID`
+
+- Example: `@System.String`
+- Example: `[String class](xref:System.String)` 
+
+For more information about using this notation, see [Using cross reference](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#using-cross-reference).
+
+> Right now, there is no easy way to find the UIDs. The best way to find the UID for an API is to search for it in this repo: [docascode/coreapi](https://github.com/docascode/coreapi). We're working on having a better system in the future.
+
+## Lists
+
+### Ordered lists
+
+1. This 
+1. Is
+1. An
+1. Ordered
+1. List  
+
+
+#### Ordered list with an embedded list
+
+1. Here
+1. comes
+1. an
+1. embedded
+    1. Miss Scarlett
+    1. Professor Plum
+1. ordered
+1. list
+
+
+### Unordered Lists
+
+- This
+- is
+- a
+- bulleted
+- list
+
+
+##### Unordered list with an embedded list
+
+- This 
+- bulleted 
+- list
+    - Mrs. Peacock
+    - Mr. Green
+- contains  
+- other
+    1. Colonel Mustard
+    1. Mrs. White
+- lists
+
+
+## Horizontal rule
+
+---
+
+## Tables
+
+| Tables        | Are           | Cool  |
+| ------------- |:-------------:| -----:|
+| col 3 is      | right-aligned | $1600 |
+| col 2 is      | centered      |   $12 |
+| col 1 is default | left-aligned     |    $1 |
+
+You can use a [Markdown table generator tool](http://www.tablesgenerator.com/markdown_tables) to help creating them more easily. 
+
+## Code
+
+### Code blocks with language identifier
+
+Use three backticks (\`\`\`) + a language ID to apply language-specific color coding to a code block. Here is the entire list of [GFM language IDs](https://github.com/jmm/gfm-lang-ids/wiki/GitHub-Flavored-Markdown-(GFM)-language-IDs).
+
+##### C&#9839;
+
+```c#
+using System;
+namespace HelloWorld
+{
+    class Hello 
+    {
+        static void Main() 
+        {
+            Console.WriteLine("Hello World!");
+
+            // Keep the console window open in debug mode.
+            Console.WriteLine("Press any key to exit.");
+            Console.ReadKey();
+        }
+    }
+}
+```
+#### Python
```

Comment:
It's kinda odd to see Python in the style guide and not VB and F#. I'm guessing that this was copied from elsewhere. Seems to we should take the extra step, delete the python example (I don't think we'll ever have any of these) and add VB and F#.